src/lru.mli

```diff
@@ -65,8 +66,8 @@ end) : sig
           else if old t.age k' < old t.age k then old t.age k' + 1
           else old t.age k' *)
 
-  val remove : 'a t -> K.t -> unit
-  (*@ remove t k
+  val remove : 'a t -> key -> unit
+  (*@ c = remove t k
```

Comment:
Any reason you need to change this?

Reply:
Not needed, but it was clearer than just K.t because we have the type key, you might as well use it !

Reply:
I agree! I was referring to the `c =` in the spec :)